HousingSearchApi/V1/Infrastructure/Factories/AssetQueryGenerator.cs

```diff
@@ -79,6 +79,7 @@ public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescripto
                         .WithMultipleFilterQuery(assetListAllRequest.IsTemporaryAccomodation, new List<string> { "assetManagement.isTemporaryAccomodation" })
                         .WithMultipleFilterQuery(assetListAllRequest.ParentAssetId, new List<string> { "rootAsset" })
                         .WithMultipleFilterQuery(assetListAllRequest.IsActive, new List<string> { "isActive" })
+                        .WithMultipleFilterQuery(assetListAllRequest.IsApproved, new List<string> { "assetContract.isApproved" })
```

Comment:
Would be good to add e2e test to check that the `isApproved` flag has the desired effect on the results.

Reply:
E2E test added